examples/wasm_node/test/test_multisig.js

```diff
@@ -232,3 +232,15 @@ describeCall('cancelMultisig', function() {
     assert(JSON.parse(signature).Signature)
   })
 })
+
+describeCall('serializeParams reported cases', function() {
+  it('newThreshold', function () {
+    const innerParams = {
+      NewTreshold: 2
```

Comment:
There is a typo here
```suggestion
      NewThreshold: 2
```

Reply:
I wonder how/why the test passes with that typo there...

Reply:
yeah that's really weird I didn't notice

Reply:
Apparently I am really consistent in my typos :
https://github.com/Zondax/filecoin-signing-tools/blob/master/signer/src/api.rs#L366